src/content/docs/en/reference/api-reference.mdx

```diff
@@ -997,15 +997,15 @@ Get the URL of the current page (useful for canonical URLs).
 **Type:** `string | undefined`
 </p>
 
-Get the URL of the previous page (will be `undefined` if on page 1).
+Get the URL of the previous page (will be `undefined` if on page 1). If using a base path, prepend the base path to the URL.
```

Comment:
```suggestion
Get the URL of the previous page (will be `undefined` if on page 1). If a value is set for [`base`](/en/reference/configuration-reference/#base), prepend the base path to the URL.
```
I like having a link to the actual value here! (Wording was fine, but with a link to value, felt more helpful to say "a value set for" here)